mzLib/Proteomics/ProteolyticDigestion/PeptideWithSetModifications.cs

```diff
@@ -617,48 +613,6 @@ public void FragmentInternally(DissociationType dissociationType, int minLengthO
             }
         }
 
-        public virtual string EssentialSequence(IReadOnlyDictionary<string, int> modstoWritePruned)
```

Comment:
did this move

Reply:
Moved to BioPolymerWithSetModsExtensions